### DISCUSSION TITLE
Micro Recording Crashes On Higher Than Program RAM Targets Machines

### DISCUSSION THREAD

**Discussion by TEOL2023** [audacity/audacity#7271] [Development]
Now that the major recording crashes are supposedly fixed I don't know yet myself as my hissy jack needs an OpenVinu out before I can update now I'm eying up those micro crashes to be fixed as well.

You know the one where you have an 8GB RAM system 4 times the amount of target RAM of 2GB yet you turn your mouse off and leave it to record something but you see a load of annoying data loss circles down below your track after you flipped your power screensaver back up to see your recording the same thing that reproduced the full program recording crash on a long recording for 2 versions straight on 00 and 01 of 3.6.

This micro crash had happened since the 3.4 series for me and hasn't been attempted to be fixed since although the full crash was more important to fix first.

Even if it requires Audacity to dump HD and HD0 files into the same folder as your track like Magix does to pull this fix off I'd still prefer annoying mini hard drive files over micro crash data loss on a long non mouse moving mouse switched off recording which makes power screensavers happen and Norton turn on.

Magix still records even when Norton and screensavers are up with the mouse switched to off why can't I demand that out of 2GB RAM Audacity on an 8GB RAM machine as well.

Yeah, the annoying micro crash which doesn't even happen on my backup recording in Magix is the next recording bug I'd like fixed as then I wouldn't need to have a second more reliable recording happen as Audacity would be the reliable recording workhouse again.

**Comment by LWinterberg** [comment]
those skips happen whenever the recording fails for a bit. This can indicate a bottleneck in some areas, such as CPU and hard drive. If it doesn't happen at a lower sample rate and depth, it may be the drive, if it doesn't happen when zoomed further out, it may be the CPU.

**Comment by TEOL2023** [comment]
![Untitled](https://github.com/user-attachments/assets/a3428623-f516-4bb9-a4f9-bac08e6f11f5)

8GB+ RAM Micro dropouts recording crashes are still in effect since 3.6.2.

Replication: Turning wireless mouse off then leaving to record in a test run for later. Whilst the program didn’t crash in testing the micro crash on high RAM machines are still happening.

Time on clock: 16:45 after turning mouse off and leaving it to record.

Loss of Data: About 3-5 seconds of blank where it’s supposed to be music.

Device: Loopback pure digital data recorder.

![Untitled1](https://github.com/user-attachments/assets/fea76190-e338-4707-9344-ecd4bbae441d)

As you can see statistics are not a problem it's quite a powerful base machine I have now and Magix doesn't do these dropouts with it's HD0 files alongside the proper ones.

This never happens with mouse on and moving about only when turning the mouse off then leaving the machine.

It might be a legit odd bug issue relating to when there is an idol switched off mouse on the program.
